package.json

```diff
@@ -15,5 +15,8 @@
   "bugs": {
     "url": "https://github.com/all-rit/AccessibilityLab1/issues"
   },
-  "homepage": "https://github.com/all-rit/AccessibilityLab1#readme"
+  "homepage": "https://github.com/all-rit/AccessibilityLab1#readme",
```

Comment:
Is there a reason for this change, or was it committed by accident

Reply:
I'm not sure how that got changed, but it definitely doesn't need to be there. Good catch